xcodeproj/internal/bazel_integration_files/clang.sh

```diff
@@ -16,6 +16,12 @@ do
   *.o)
     break
     ;;
+  -v)
+    DEV_DIR_PREFIX=$(awk '{ sub(/.*-isysroot /, ""); sub(/.Contents\/Developer.*/, ""); print}' <<< "${@:1}")
+    clang="$DEV_DIR_PREFIX/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang"
```

Comment:
Can we add a TODO here about custom toolchains, similar to the one in swiftc.py?

Reply:
Makes sense. Updated!